test_conformance/math_brute_force/main.cpp

```diff
@@ -1392,17 +1392,14 @@ int InitILogbConstants( void )
     "}\n";
 
     cl_program query;
-    error = create_single_kernel_helper(gContext, &query, NULL, 1, &kernel, NULL);
-    if (NULL == query || error)
-    {
-        vlog_error( "Error: Unable to create program to get FP_ILOGB0 and FP_ILOGBNAN for the device. (%d)", error );
-        return error;
-    }
-
-    cl_kernel k = clCreateKernel( query, "GetILogBConstants", &error );
-    if( NULL == k || error)
+    cl_kernel k;
```

Comment:
Prefer `clKernelWrapper`.

Could also switch the to program to `clProgramWrapper`, but that's getting further from the scope of this PR so fine to leave that for later.